include/controller/c/webots/supervisor.h

```diff
@@ -94,7 +94,9 @@ WbNodeRef wb_supervisor_node_get_parent_node(WbNodeRef node);
 WbNodeRef wb_supervisor_node_get_selected();
 WbNodeType wb_supervisor_node_get_type(WbNodeRef node);
 WbFieldRef wb_supervisor_node_get_field(WbNodeRef node, const char *field_name);
+WbFieldRef wb_supervisor_node_get_field_by_index(WbNodeRef node, const int index);
```

Comment:
It is useless to define an argument passed by value as constant:
```suggestion
WbFieldRef wb_supervisor_node_get_field_by_index(WbNodeRef node, int index);
```

Same for `wb_supervisor_node_get_proto_field_by_index`.